Moderation-Policy.md

```diff
@@ -331,24 +331,26 @@ remove resigning team member from respective permissions and private access.
 * [ljharb](https://github.com/ljharb) -
   **Jordan Harband** &lt;ljharb@gmail.com&gt;
 
-## Escalation of Issues
+## Escalation of issues
 
-Moderation issue disputes not involving a TSC voting member or Moderation Team
-member may be escalated to the TSC for review by tagging the
-original issue, pull request, or associated nodejs/moderation repository
-tracking issue with the `moderation-review` label. Any such Moderation action
-may be overturned through a TSC vote.
+Any code of conduct report, and any decision made by the moderation team, can
+be escalated to the [OpenJS Code of Conduct Team](https://github.com/openjs-foundation/cross-project-council/blob/main/conduct/COC_POLICY.md#escalation).
```

Comment:
By the way, the CoC policy makes a distinction between escalation and appeal and your policy should address both.